### PR TITLE
[ONNX] Add support for ONNX 1.6 TopK

### PR DESCRIPTION
**Pull request by postrational**
Add support for ONNX `TopK` attributes: `largest` and `sorted`

### CODE REVIEW

File: src/ngraph/runtime/cpu/unit_test.manifest [hunk 1]
```diff
@@ -24,3 +24,6 @@ lrn_6D_across_2_axes
 
 # ONNX TopK with dynamic K
 top_k_opset_10
+
+# ONNX TopK with dynamic K
+top_k_opset_10
```
Comment (arogowie-intel) [inline src/ngraph/runtime/cpu/unit_test.manifest]: There is duplicate `top_k_opset_10`. Didn't you intended to write: `top_k_opset_11_const_k_smallest` ?

Comment (tsocha) [inline src/ngraph/runtime/cpu/unit_test.manifest]: You should remove it.

File: src/ngraph/frontend/onnx_import/op/topk.cpp [hunk 2]
```diff
@@ -79,6 +92,31 @@ namespace ngraph
                 }
             }
 
+            namespace set_11
+            {
+                NodeVector topk(const Node& node)
+                {
+                    // Process inputs
+                    auto data = node.get_ng_inputs().at(0);
```
Comment (tsocha) [inline src/ngraph/frontend/onnx_import/op/topk.cpp]:
```suggestion
                    auto data = get_data(node);
```
Is it worth to add a new function for consistency?

File: src/ngraph/frontend/onnx_import/op/topk.cpp [hunk 3]
```diff
@@ -79,6 +92,31 @@ namespace ngraph
...
+                    auto k = get_k(node);
+
+                    // Process attributes
+                    const auto axis = get_axis(node);
+                    const auto largest = node.get_attribute_value<std::int64_t>("largest", 1);
+                    const auto sorted = node.get_attribute_value<std::int64_t>("sorted", 1);
```
Comment (tsocha) [inline src/ngraph/frontend/onnx_import/op/topk.cpp]: You can read these attributes to `bool` variable.

example from `gemm` operator:
```
                    bool trans_a = node.get_attribute_value<int64_t>("transA", 0);
                    bool trans_b = node.get_attribute_value<int64_t>("transB", 0);
```

File: src/ngraph/frontend/onnx_import/op/topk.cpp [hunk 4]
```diff
@@ -20,25 +20,38 @@
...
-static ngraph::NodeVector get_outputs(const std::shared_ptr<ngraph::Node>& top_k)
+/// \return Return the second input to the TopK node reshaped to a scalar.
+static std::shared_ptr<ngraph::Node> get_k(const ngraph::onnx_import::Node& node)
 {
-    std::shared_ptr<ngraph::Node> indices =
-        std::make_shared<ngraph::op::GetOutputElement>(top_k, 0);
-    std::shared_ptr<ngraph::Node> values = std::make_shared<ngraph::op::GetOutputElement>(top_k, 1);
+    auto k_node = node.get_ng_inputs().at(1);
+    NGRAPH_CHECK(shape_size(k_node->get_shape()) == 1,
+                 "ONNX TopK operator: 'K' parameter must contain a single positive value.",
```
Comment (tomdol) [inline src/ngraph/frontend/onnx_import/op/topk.cpp]: The error message mentions the sign of a value but there's no check for a sign here.

File: src/ngraph/frontend/onnx_import/utils/reshape.cpp [hunk 5]
```diff
@@ -100,6 +99,15 @@ namespace ngraph
                              "Scalar value can't be derived from a node with ",
                              node_shape);
 
+                // If node is a Constant, recreate as Constant with Shape{}
+                if (node->is_constant())
+                {
+                    const auto value =
+                        ngraph::as_type_ptr<ngraph::op::Constant>(node)->get_data_ptr();
+                    return std::make_shared<ngraph::op::Constant>(
```
Comment (tomdol) [inline src/ngraph/frontend/onnx_import/utils/reshape.cpp]: I think this should be unit tested to verify that values in the reshaped constant propagate/broadcast correctly.

File: src/ngraph/frontend/onnx_import/op/topk.cpp (hunk 4 above)
Comment (postrational) [reply src/ngraph/frontend/onnx_import/op/topk.cpp]: The value is unknown at the time of graph import. We're only checking the shape, but I think the informative description of the expected value (from ONNX spec) may be helpful. I can remove the word `single` if you insist.

File: src/ngraph/frontend/onnx_import/op/topk.cpp (hunk 2 above)
Comment (postrational) [reply src/ngraph/frontend/onnx_import/op/topk.cpp]: I don't think so. If the function did any preparation of the input node, then yes, but in this situation I think this is clearer.

File: src/ngraph/frontend/onnx_import/op/topk.cpp (hunk 3 above)
Comment (postrational) [reply src/ngraph/frontend/onnx_import/op/topk.cpp]: I could, but we're changing this line in the `v1` upgrade this sprint.

File: src/ngraph/frontend/onnx_import/utils/reshape.cpp (hunk 5 above)
Comment (postrational) [reply src/ngraph/frontend/onnx_import/utils/reshape.cpp]: Done.

File: test/CMakeLists.txt [hunk 6]
```diff
@@ -342,9 +342,10 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     list(APPEND MULTI_TEST_SRC
             onnx/onnx_import.in.cpp
             onnx/onnx_import_convpool.in.cpp
+            onnx/onnx_import_quant.in.cpp
             onnx/onnx_import_reshape.in.cpp
             onnx/onnx_import_rnn.in.cpp
-            onnx/onnx_import_quant.in.cpp)
+            onnx/onnx_import_utils_reshape.in.cpp)
```
Comment (tsocha) [inline test/CMakeLists.txt]: Could you sort this list?

**Pull request merged**